cms/djangoapps/export_course_metadata/toggles.py

```diff
@@ -0,0 +1,16 @@
+"""
+Toggles for export_course_metadata app
+"""
+
+from edx_toggles.toggles import WaffleFlag
+
+# Waffle flag to split library to new view.
+# .. toggle_name: export_course_metadata
+# .. toggle_implementation: WaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Export of course metadata (initially to s3 for use by braze)
```

Comment:
I wouldn't bother mentioning braze in open edx code, but this is a comment on a temporary waffle, so who cares.